crates/bevy_text/src/text2d.rs

```diff
@@ -211,12 +211,12 @@ pub fn update_text2d_layout(
                     // queue for further processing
                     queue.insert(entity);
                 }
-                Err(e @ TextError::FailedToAddGlyph(_)) => {
+                Err(e @ TextError::FailedToAddGlyph(_) | e @ TextError::FailedToAcquireMutex) => {
```

Comment:
These patterns can be simplified to `Err(e @ (TextError::FailedToAddGlyph(_) | TextError::FailedToAcquireMutex))`.